cypress/integration/group3/organizations.ts

```diff
@@ -260,13 +260,72 @@ describe('Dockstore Organizations', () => {
 
   describe('Verify title tags ', () => {
     it('Specific organization', () => {
-      cy.visit('/organizations/1');
+      cy.visit('/organizations/Potatoe');
       cy.title().should('eq', 'Dockstore | Organization');
     });
 
     it('Collection', () => {
-      cy.visit('/organizations/1/collections/1');
+      cy.visit('/organizations/Potatoe/collections/veryFakeCollectionName');
       cy.title().should('eq', 'Dockstore | Collection');
     });
   });
+
+  describe('Find organization and collection by alias', () => {
+    it('organization alias', () => {
+      cy.server();
+      cy.route({
+        url: '/organizations/fakeAlias/aliases',
```

Comment:
So it I understand a visit to `/aliases/organizations/fakeAlias` should redirect to `Cypress.config().baseUrl + '/organizations/Potatoe`. But what role is `/organizations/fakeAlias/aliases` playing here?

Reply:
This is the API call being used to get an organizations alias called fakeAlias, this is not what is seen in the browser URL.